controllers/controllers.js

```diff
@@ -7,12 +7,30 @@ exports.getAllEndpoints = (req, res, next) => {
 
 exports.getAllTopics = (req, res, next) => {
     selectAllTopics()
-    .then(({rows}) => {
+    .then(({ rows }) => {
         const topics = rows
 
         res.status(200).send({topics})
     })
     .catch((err) => {
         next(err)
     })
+}
+
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params
+    selectArticleById(article_id)
+    .then(({ rows }) => {
+        if (rows.length === 0) {
+            const err = {
+                status: 404,
+                msg: 'Invalid article_id'
+            }
+            next(err)
+        } else {
+            const article = rows[0]
```

Comment:
our model can send back rows[0] so the controller only needs to send this back to the client